src/helpers/ComponentRegister.ts

```diff
@@ -100,8 +97,17 @@ export class ComponentRegister {
    * });
    * ```
    *
-   * @param options - Options to customize the underlying components used by
-   * library components.
+   * ```ts
+   * // Using `Link` from React Router as `LinkButton`.
+   * ComponentRegister.init({
+   *   components.
+   *   LinkButton: ({id, className, href, children }) => (
+   *     <ReactRouter.Link id={id} className={className} to={href}>
+   *       {children}
+   *     </ReactRouter.Link>
+   *   )
+   * });
+   * ```
```

Comment:
We could probably do with an example that isn't a repeat of the `Link` example above to demonstrate why it might need to be different.